imports/plugins/core/accounts/server/i18n/en.json

```diff
@@ -21,7 +21,8 @@
         },
         "groupsInvite": {
           "name": "John Smith",
-          "email": "johnsmith@reactioncommerce.com"
+          "email": "johnsmith@reactioncommerce.com",
```

Comment:
We really shouldn't hardcode examples like this, with @reactioncommerce, it's better to use @localhost if you need to.  We don't know where this UI will end up being used, and it's better to be general.